client/src/extension.ts

```diff
@@ -584,8 +591,7 @@ function considerStartingBackend(backendName: string) {
             backend: backendName,
             manuallyTriggered: true,
             forceRestart: false,
-            isViperServerEngine: false //TODO: how to set that correctly
-
+            isViperServerEngine: true
```

Comment:
This option is not necessary anymore as the concept of ViperServerEngine is obsolete. If it's easy to carve out the code that mentioned this concept, please do so; otherwise, add an explaination in the comments.